src/main/java/ee/pardiralli/components/ScheduledTasks.java

```diff
@@ -0,0 +1,26 @@
+package ee.pardiralli.components;
+
+import ee.pardiralli.service.RaceService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@AllArgsConstructor(onConstructor = @__(@Autowired))
+@Slf4j
+public class ScheduledTasks {
+    private final RaceService raceService;
+
+    /**
+     * Every day at 00 check for races to close
+     */
+    @Scheduled(cron = "0 0 00 * * *")
```

Comment:
Should be
@Scheduled(cron="5 0 0 * * *", zone="Europe/Athens")
to run 5 seconds after midnight (just in case of inaccuracies) in Eastern European time zone